### PR TITLE
Use Log in place of Timber in MainApplication

### LINKED ISSUE #1425: Crash in the Chat Activity in the release apk
**Actual Behaviour**

On logging in, the app crashes on the chat screen.

**Expected Behaviour**

The app should not crash.

**Steps to reproduce it**

Log in to the app and wait for the chat screen to open

**LogCat for the issue**

**Would you like to work on the issue?**

Yes me and @arundhati24

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1425 and #1426 

Changes: Replaced timber with log in MainApplication.java file

### CODE REVIEW

**Comment by batbrain7** [comment]
@iamareebjamal please review

File: app/src/main/java/org/fossasia/susi/ai/MainApplication.java [hunk 1]
```diff
@@ -68,11 +68,13 @@ protected void log(int priority, String tag, @NonNull String message, Throwable
                 return;
             }
 
+            Timber.log(priority, tag, message, throwable);
+
             if (priority == Log.ERROR) {
                 if (throwable == null) {
-                    Timber.e(message);
+                    Log.e("MainApplication", message);
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/MainApplication.java]: Replace String with TAG.

File: app/src/main/java/org/fossasia/susi/ai/MainApplication.java [hunk 2]
```diff
@@ -68,11 +68,13 @@ protected void log(int priority, String tag, @NonNull String message, Throwable
...
                 } else {
-                    Timber.e(throwable, message);
+                    Log.e("MainApplication", throwable.toString());
                 }
```
Comment (arundhati24) [inline app/src/main/java/org/fossasia/susi/ai/MainApplication.java]: Replace String with TAG.

**Comment by batbrain7** [comment]
@arundhati24 done the changes

File: app/src/main/java/org/fossasia/susi/ai/MainApplication.java [hunk 3]
```diff
@@ -68,11 +68,13 @@ protected void log(int priority, String tag, @NonNull String message, Throwable
                 return;
             }
 
+            Timber.log(priority, tag, message, throwable);
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/MainApplication.java]: Again a recursive call

File: app/src/main/java/org/fossasia/susi/ai/MainApplication.java [hunk 4]
```diff
@@ -70,9 +70,9 @@ protected void log(int priority, String tag, @NonNull String message, Throwable
 
             if (priority == Log.ERROR) {
                 if (throwable == null) {
-                    Timber.e(message);
+                    Log.e(TAG, message);
                 } else {
-                    Timber.e(throwable, message);
+                    Log.e(TAG, throwable.toString());
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/MainApplication.java]: You should also log message

File: app/src/main/java/org/fossasia/susi/ai/MainApplication.java [hunk 5]
```diff
@@ -72,7 +72,7 @@ protected void log(int priority, String tag, @NonNull String message, Throwable
                 if (throwable == null) {
                     Log.e(TAG, message);
                 } else {
-                    Log.e(TAG, throwable.toString());
+                    Log.e(TAG, message + throwable.toString());
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/MainApplication.java]: Please think about how the message needs to be formatted

`message + "\n" + throwable.toString()`

**Pull request merged**